vaultpress.php

```diff
@@ -20,26 +20,30 @@
  * We want to fail gracefully if `composer install` has not been executed yet, so we are checking for the autoloader.
  * If the autoloader is not present, let's log the failure, pause VaultPress, and display a nice admin notice.
  */
-$loader = dirname( __FILE__ ) . '/vendor/autoload.php';
-if ( is_readable( $loader ) ) {
-	require $loader;
+$autoloader = dirname( plugin_basename( __FILE__ ) ) . 'vendor/autoload_packages.php';
+if ( is_readable( $autoloader ) ) {
+	require $autoloader;
 } else {
+	if ( defined( 'WP_DEBUG' ) && WP_DEBUG ) {
+		error_log(
+			__( 'Your installation of VaultPress is incomplete. If you installed it from GitHub, please run <code>composer install</code>.', 'vaultpress' )
```

Comment:
```suggestion
			esc_html__( 'Your installation of VaultPress is incomplete. If you installed it from GitHub, please run <code>composer install</code>.', 'vaultpress' )
```

Reply:
Although `wp_kses` usually isn't used for escaping, in our case it should do the job well, given that we want to localize this string 👍